app/views/notes/show.html.erb

```diff
@@ -1,3 +1,16 @@
+
+<div class="row">
+  <div class="card file-card col m8 offset-m2">
+      <div class="card-content white-text">
+        <h3 class="card-title">Note:</h3>
+      <div>Name: <%= note.note_with_type %></div>
+      <div>Author: <%= note.user_id %></div>
+      <div>Category: <%= note.category.name %></div>
+      <div>Body: <%= note.body %></div>
+    </div>
+  </div>
+</div>
+
```

Comment:
^ this should only be shown if authorized (line 2 through line 12) - it looks like you have some duplication